uxarray/grid.py

```diff
@@ -379,6 +384,100 @@ def integrate(self, var_ds, quadrature_rule="triangular", order=4):
 
         return integral
 
+    # A one-time-used helper function that will only be used in the build_face_edges_connectivity map() function usage
+    def __replace_fill_val(self, row, index, rep_val):
+        row[index] = rep_val
+        return row
+
+    def build_face_edges_connectivity(self):
+        """A DataArray of indices indicating edges that are neighboring each
+        face.
+
+        Notes
+        -----
+        This function will add `Grid.ds.Mesh2_face_edges` to the `Grid` class, which is an integer
+        DataArray of size (nMesh2_face, MaxNumNodesPerFace)
+        """
+        mesh2_face_nodes = self.Mesh2_face_nodes.values
```

Comment:
No changes needed, but for future reference with #241 merged, we can guarantee that `self.Mesh2_face_nodes` will always have `dtype=INT_DTYPE` and `_FillValue=INT_FILL_VALUE`